README.md

```diff
@@ -139,6 +139,7 @@ Prepare database once before you start by running the following command in a sec
 ```bash
 # in main folder while docker-compose is up
 $ docker-compose exec backend yarn run db:migrate init
+$ docker-compose exec backend yarn run db:migrate up
```

Comment:
Without dash is the modern command, see:
https://docs.docker.com/compose/reference/

```suggestion
$ docker compose exec backend yarn run db:migrate init
$ docker compose exec backend yarn run db:migrate up
```